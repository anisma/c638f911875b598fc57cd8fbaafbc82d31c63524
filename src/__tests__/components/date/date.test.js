import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import PropTypes from 'prop-types';
import Date from '../../../components/date/date.component'

describe('Date Component', () => {
  it('renders correctly enzyme', () => {
  const wrapper = shallow(<Date />)

  expect(toJson(wrapper)).toMatchSnapshot();
  });
   
   it('should have correct props', () => {
      Date.propTypes = {
         day: PropTypes.string.isRequired,
         date: PropTypes.string.isRequired,
         hidden: PropTypes.bool.isRequired,
         active: PropTypes.bool.isRequired
      };
      const wrapper = shallow(<Date day='Senin' date='23' active={false} hidden={true} />);
       expect(wrapper.props().day).toEqual('Senin');
       expect(wrapper.props().date).toEqual('23');
       expect(wrapper.props().hidden).toEqual(true);
       expect(wrapper.props().active).toEqual(false);
   })
});