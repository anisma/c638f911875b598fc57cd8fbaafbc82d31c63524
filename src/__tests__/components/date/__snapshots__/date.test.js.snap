// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Date Component renders correctly enzyme 1`] = `
<styled.span>
  <styled.span />
  <styled.span />
</styled.span>
`;
